broadinstitute/ddp-study-server
# DSP-7581: Merging RC export work back into develop

This merges all of the previously approved changes on the RC branch relating to export work.  It also includes an update to correctly handle file uploads in the export

TO TEST:
1. go to participant list for Singular
2. click download, and then download with the default options
3. confirm download does not error
4. review downloaded spreadsheet, confirm that the field "singularNotes" is populated for the participant PPW7EY with the text "enrolled at last"
